### PR TITLE
🐳 Update likecoin env

### PR DESCRIPTION
**Pull request by inDream**
Connects #31 
Connects #54 
Connects #86

### CODE REVIEW

File: docker-compose.testing.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ services:
   nuxt:
     environment:
       API_HOST: http://nuxt:3000
+      LIKECOIN_AUTH_URL:
```
Comment (nwingt) [inline docker-compose.testing.yml]: Missing ?

Comment (inDream) [reply docker-compose.testing.yml]: Need to wait for user login integration test.

File: docker-compose.yml [hunk 2]
```diff
@@ -18,6 +18,9 @@ services:
       HOST: 0.0.0.0
       IPFS_HOST: http://ipfs:5001
       IPFS_MASTER: http://ipfs:5001
+      LIKECHAIN_HOST: ${LIKECHAIN_HOST}
+      LIKECOIN_AUTH_URL: https://api.rinkeby.like.co/api/users/challenge
+      SIG_ADDRESS: ${SIG_ADDRESS}
```
Comment (rickmak) [inline docker-compose.yml]: What is `SIG_ADDRESS`?

I cannot find any code using it.

**Comment by rickmak** [comment]
Merge via cf082c0

**Pull request closed without merging**